### PR TITLE
[rjsf] some small fixes 

### PR DESCRIPTION
**Pull request by harkiratsm**
Signed-off-by: harkiratsm <multaniharry714@gmail.com>

**Description**

This PR fixes #

- Added ellipsis property to Input label 
- prevent user to enter negative value 
- 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomUpDownWidget.js [hunk 1]
```diff
@@ -25,8 +25,8 @@ const CustomUpDownField = (props) => {
         key={props.id}
         value={props?.value}
         variant="standard"
-        onChange={e => props?.onChange(e.target.value)}
-        type="number"
+        onChange={e => e.target.value<0?props?.onChange(e.target.value=""):props?.onChange(e.target.value)}
+        type="tel"
```
Comment (leecalcote) [inline ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomUpDownWidget.js:29]: Curious: what is a type "tel"? Telephone number?

Comment (harkiratsm) [reply ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomUpDownWidget.js:29]: The reason to remove the "number" was to remove the painpoints that user experience with the input field with that updown  arrow , which probably noone cares to click .

image

Comment (leecalcote) [reply ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomUpDownWidget.js:29]: Use of the up/down arrows is option, right? Users have the choice of up/down arrows or entering a number. It's good to have this choice.

Comment (leecalcote) [reply ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomUpDownWidget.js:29]: Thank you for this screenshot. It's helpful.

Comment (leecalcote) [inline ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomUpDownWidget.js:29]:
```suggestion
        type="number"
```

Comment (leecalcote) [reply ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomUpDownWidget.js]: We don't want the user to click the down arrow, but removing the arrow doesn't prevent them from entering a negative number either.

Changing the field type is not a good solution and likely to cause issues - "tel" is generally, not a type that we'll be concerned with for a couple years.

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
But I can be a challenge, but please do try to avoid a description like "some small fixes". I'm not being picky, but I'm trying to help streamline reviews.

**Pull request merged**